Added package metadata to controls project to enable publishing (#16611)
files-community/Files: src/Files.App.Controls/README.md

Reviewer: Where is the readme displayed (aside from on GitHub)?

Author: On NuGet
e.g. https://www.nuget.org/packages/Crimson

> These controls are provided "as is", with no guaranteed support, but we hope they prove useful to the developer community.
# 📁 Files UI Controls

##### Elevate your WinUI applications with our collection of custom-built controls, crafted specifically to address our needs in Files.

---

This package contains various controls for the [Files app](https://files.community), including `ThemedIcon`, `Toolbar` and various storage controls.
It is available [on NuGet](https://www.nuget.org/packages/Files.App.Controls), however is unlisted and can only be installed by manually typing the package name and version number into the MSBuild project file.

> Please do not open issues on the Files repository about this package, as it is not officially supported by the Files team and is kept online for reference.

Reviewer: ```suggestion
Please do not open issues on the Files repository about this package, as it is not officially supported by the Files team and is provided as is.
```

> > These controls are provided "as is," with no guaranteed support, but we hope they prove useful to the developer community.

Reviewer: ```suggestion
> These controls are provided "as is", with no guaranteed support, but we hope they prove useful to the developer community.
```

>  

Author: ```suggestion

```
